planet4-gutenberg-blocks.php

```diff
@@ -439,6 +439,44 @@ function ( $std_class_object ) {
 	],
 ];
 
+add_filter(
+	'render_block_core/query-pagination-previous',
+	function ( $content, $parsed, $block ) {
+		if ( empty( $block->attributes['label'] ) ) {
+			$block->attributes['label'] = __( 'Prev', 'planet4-blocks' );
+			return $block->render();
+		}
+
+		// Check if the button isn't rendered, then return it.
+		if ( empty( $content ) ) {
+			return '<a href="/" class="wp-block-query-pagination-previous disabled">' . __( 'Prev', 'planet4-blocks' ) . '</a>';
```

Comment:
This link would lead to the homepage, if we want no link we could remove the href attribute for example ?

Reply:
Yep! In fact, I've added the disable class to disable the link but I've just noticed that I forgot to add that rule 🤦